clang/test/CIR/Translation/zeroinitializer.cir

```diff
@@ -4,22 +4,22 @@
 module {
```

Comment:
Do we need the test or I should remove the entire file?

Reply:
I think it makes sense to remove the whole file, thanks for noticing it!

Reply:
done